charts/spire/charts/spire-agent/README.md

```diff
@@ -88,6 +88,7 @@ A Helm chart to install the SPIRE agent.
 | `workloadAttestors.unix.enabled`                    | Enables the Unix workload attestor                                                                                                                                                                | `false`                                                                          |
 | `workloadAttestors.k8s.enabled`                     | Enables the Kubernetes workload attestor                                                                                                                                                          | `true`                                                                           |
 | `workloadAttestors.k8s.skipKubeletVerification`     | If true, kubelet certificate verification is skipped                                                                                                                                              | `true`                                                                           |
+| `workloadAttestors.k8s.kubeletCaPath`               | Path to the file containing CA certificates used to verify the kubelet certificate                                                                                                                | `/run/secrets/kubernetes.io/serviceaccount/ca.crt`                               |
```

Comment:
This doesn't follow the documented naming convention.
It should be kubeletCAPath

I'm not seeing where it is being mounted into the container too. Are we messing some code?

Reply:
Oh, its just assuming that its already there, and its defaulting to the apiserver's CA. In vanilla Kubernetes, that is not the case. :/